src/main/java/com/google/maps/OkHttpRequestHandler.java

```diff
@@ -102,11 +102,9 @@ public void setQueriesPerSecond(int maxQps) {
     rateLimitExecutorService.setQueriesPerSecond(maxQps);
   }
 
-  @Override
+  @Override @Deprecated
   public void setQueriesPerSecond(int maxQps, int minimumInterval) {
-    dispatcher.setMaxRequests(maxQps);
-    dispatcher.setMaxRequestsPerHost(maxQps);
-    rateLimitExecutorService.setQueriesPerSecond(maxQps, minimumInterval);
+    this.setQueriesPerSecond(maxQps);
```

Comment:
Can you log a warning or something here? I'm concerned that ignoring an explicitly provided parameter will lead to unexpected behaviour for users.